src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,4 +49,17 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code consultation} for display to the user.
+     */
+    public static String format(Consultation consultation) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(consultation.getDate())
+                .append("; Date: ")
+                .append(consultation.getTime())
+                .append("; Time: ");
```

Comment:
this seems to be in the wrong order, appears that you would put the date first, followed by its label e.g.

returns `2022-03-01; Date: 11:30; Time: ...`